data-model/SDP-data-model-templates/textual-artifact_sdp-template.json

```diff
@@ -7,6 +7,20 @@
   ],
   "extent": "",
   "embodies": "URI of Work Authority File",
+  "attested_title": {
```

Comment:
I realized that we are having RAs grab this data but we didn't have anywhere in the JSON to put it...We just had the rubric paratext, which is sometimes more info than we want for the title.